docs/ref/filters.txt

```diff
@@ -3,6 +3,118 @@ Filter Reference
 
 This is a reference document with a list of the filters and their arguments.
 
+.. _core-arguments:
+
+Core Arguments
+--------------
+
+The following are the core arguments that apply to all filters.
+
+``name``
+~~~~~~~~
+
+The name of the field this filter is supposed to filter on, if this is not
+provided it automatically becomes the filter's name on the ``FilterSet``.
+You can traverse "relationship paths" using Django's ``__`` syntax to filter
+fields on a related model. eg, ``manufacturer__name``.
+
+``label``
+~~~~~~~~~
+
+The label as it will apear in the HTML, analogous to a form field's label
+argument.
+
+``widget``
+~~~~~~~~~~
+
+The django.form Widget class which will represent the ``Filter``.  In addition
+to the widgets that are included with Django that you can use there are
+additional ones that django-filter provides which may be useful:
+
+    * ``django_filters.widgets.LinkWidget`` -- this displays the options in a
+      manner similar to the way the Django Admin does, as a series of links.
+      The link for the selected option will have ``class="selected"``.
+
+``action``
```

Comment:
I agree, but think it should be handled in a separate PR as it's backwards incompatible.